kedro/templates/project/hooks/utils.py

```diff
@@ -0,0 +1,150 @@
+from pathlib import Path
+import shutil
+import click
+
+current_dir = Path.cwd()
+
+lint_requirements = "black\nruff\n"
```

Comment:
I think we should also pin `black` and `ruff` just like the `pytest` versions below so there's consistency between all dependencies.